src/app/summary/services/summary.module.ts

```diff
@@ -0,0 +1,34 @@
+import {NgModule} from '@angular/core';
+import {CommonModule} from '@angular/common';
+import {ChartComponent} from '../components/chart/chart.component';
+import {MiniCardComponent} from '../components/mini-card/mini-card.component';
+import {SummaryComponent} from '../pages/summary/summary.component';
+import {SummaryService} from './summary.service';
+import {FaIconLibrary, FontAwesomeModule} from '@fortawesome/angular-fontawesome';
+import {fas} from '@fortawesome/free-solid-svg-icons';
+import {ChartsModule} from "ng2-charts";
+
+@NgModule({
+            declarations: [
+              ChartComponent,
+              MiniCardComponent,
+              SummaryComponent,
+            ],
+            imports: [
+              CommonModule,
+              ChartsModule,
+              FontAwesomeModule,
+            ],
+            exports: [
+              ChartComponent,
+              MiniCardComponent,
+              SummaryComponent,
+            ],
+            providers: [SummaryService]
```

Comment:
You don't need to add services in the provider. Singleton services are registered on 'root' within the service decorator and non-singleton services are registered in the component they're used in. See https://github.com/ebi-ait/projects-index/blob/master/src/app/projects/pages/projects-list/projects-list.component.ts#L19